### ISSUE TITLE
[Feature] Run multiple validation sets during training

### ISSUE THREAD

**Issue by shani-sony** [open-mmlab/mmengine#1406]
### What is the feature?

I want to be able to run multiple validation sets during training of `mmdetection`. I made some modifications in `loops.py` and other files to read multiple dataloaders but I'm still having issues with the evaluator and metric files because each evaluator's `ann_file` has to match the right dataset. Any ideas?
Thanks.

**Comment by HAOCHENYE** [comment]
https://github.com/open-mmlab/mmocr/blob/main/mmocr/evaluation/evaluator/multi_datasets_evaluator.py

Hope it helps :smiley:

**Comment by shani-sony** [comment]
I had to modify it a little bit but it actually helped! Thank you :)
